docs/spec/index.html

```diff
@@ -1,740 +1,952 @@
           </li>
           <li>
-            <em>Linked Data</em> establishes concise semantic context of the
-            data exchanged, enabled with JSON-LD by the trace-vocab
-            specification.
+            `TraceablePresentation.workflow.definition` — The ID for the
+            workflow definition, or the <em>type</em> of workflow that
+            the presented credentials belong to. This ID MUST be a UUID
+            v4 per [[rfc4122]].
           </li>
           <li>
-            <em>Semantics</em> are defined in a multitude of established
-            vocabularies and come in many forms, from narrowly specialized,
-            industry-vertical vocabularies, to broadly-applicable, generic
-            vocabularies.
+            `TraceablePresentation.workflow.instance` — The ID for the
+            unique instance of the workflow, e.g., party A sending goods
+            of a certain type (with particular data requirements to meet
+            regulatory needs) to party B using workflow type X as
+            defined by `TraceablePresentation.workflow.definition`. This
+            ID MUST be a UUID v4 per [[rfc4122]].
           </li>
         </ul>
-      </section>
+        <p>
+          `TraceablePresentation.id` <em>MUST</em> be unique to each
+          presentation. A presentation received with a duplicate ID
+          <em>MUST</em> be rejected with a 409 Conflict status code per
+          [[rfc9110]]. A presenter encountering this error,
+          <em>SHOULD</em> generate a new presentation, with a new UUID
+          v4 for the ID and then retry the presentation.
+        </p>
+        <p class="issue">
+          There is active discussion in
+          <a href="https://github.com/w3c-ccg/traceability-interop/issues/485">Issue
+            485</a>
+          regarding the addition of a member to a
+          <a
+            href="https://w3c-ccg.github.io/traceability-vocab/#TraceablePresentation">Traceable
+            Presentation</a>
+          with the name `replacesPresentation`. If this member is
+          present, its value <em>SHOULD</em> be an ID of a prior
+          presentation and <em>SHOULD</em> act as a signal to replace
+          the prior presentation and any credentials contained within,
+          and to prefer credentials in the new presentation.
+        </p>
+        <p>
+          To assemble a collection of Credentials linked together by a
+          workflow, data may be queried by a combination of
+          `TraceablePresentation.workflow.definition` and
+          `TraceablePresentation.workflow.instance`, assuming that the
+          implementation's underlying data store persists presentation
+          metadata upon receipt. Rules for ensuring the retrieval of the
+          most "correct" credentials associated for that workflow
+          instance are covered below in the section on Credentials.
+        </p>
 
+        <p class="issue">
```

Comment:
convert to an issue that will be groomed by our github process.

Reply:
Issue #496 has been created to track this

Reply:
Make sure to add, https://respec.org/docs/#data-number

Once its added, the comment will be automatically marked out of date, and can be resolved.